Prog/Eviction Group/DC-evic-analysis-need.R

```diff
@@ -36,50 +34,92 @@ clean_pums <- dc_pums_22 %>%
     GRNTP = GRNTP*(ADJHSG/1000000), # adjusting to match years prior to 2022 to 2022 $ value for gross rent and household income 
     HINCP = HINCP*(ADJINC),
     inc_month_30 = (HINCP*0.30)/12, # 30% of monthly income 
+    inc_month_50 = (HINCP*0.50)/12, # 50% of monthly income
     rent_burden = case_when(
       inc_month_30 <= GRNTP ~ "Rent Burden", # If 30% of monthly income less than or equal to gross monthly rent cost, HH is rent burden 
       HINCP <= 0 & GRNTP > 0 ~ "Rent Burden", # If HH income is less than or equal to 0 and the HH pays rent, HH is rent burden
-      TRUE ~ "Not Rent Burden")) %>% # All else households not rent burden 
-  dmv_hh_inc() # to have HH income bins from function in DC-HH-inc-cat.R
+      TRUE ~ "Not Rent Burden"), # All else households not rent burden
+    severe_rent_burden = case_when(
+      inc_month_50 <= GRNTP ~ "Severe Rent Burden", 
+      HINCP <= 0 & GRNTP > 0 ~ "Severe Rent Burden", # If HH income is less than or equal to 0 and the HH pays rent, HH is rent burden
+      TRUE ~ "Not Severely Rent Burden")) %>%  
+  dmv_hh_inc() # attach HH income bins from function in DC-HH-inc-cat.R
 
 total_hh_dc <- clean_pums %>% # testing weights to ensure match to ACS 5 year data for total renter, occupied households 
-  summarize(total_hh = sum(WGTP)) # 184,459 matches ACS 5 year (184,920) 
+  summarise(total_hh = sum(WGTP)) # 184,459 matches ACS 5 year (184,920) 
 
 rentburden_inccat <- clean_pums %>% # low income renters by AMI levels and rent burden 
   filter(inc_cat == "below 30 AMI" | inc_cat == "30_40AMI" | inc_cat == "40_50AMI") %>% 
   group_by(inc_cat, rent_burden) %>%
-  summarize(Total = sum(WGTP)) %>% #summing by HH weight
+  summarise(Total = sum(WGTP)) %>% #summing by HH weight
   pivot_wider(names_from = inc_cat, values_from = Total) %>%
   bind_rows(summarise(., across(where(is.numeric), sum), 
                       across(where(is.character), ~'Total'))) %>% #creating total row 
   select(rent_burden, `below 30 AMI`, `30_40AMI`, `40_50AMI`) %>% #rearranging columns
   mutate_if(is.numeric, round, -2)
 
```

Comment:
May be helpful to comment that the `mutate_if()` is rounding numeric variables to the nearest hundred. Not sure if that's an immediate takeaway when folks see -2.